grants_tagger_light/bertmesh/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .model import BertMesh, BertMeshHFCompat
```

Comment:
As discussed in slack, let's remove this folder and incorporate any needed functionality elsewhere